### ISSUE TITLE
No pytest topology mark in VxLAN-CRM testcase

### ISSUE THREAD

**Issue by echuawu** [sonic-net/sonic-buildimage#15331]
#### Description

There is no pytestmark specified in tests/vxlan/test_vnet_route_leak.py, which leads to check failure when running this script.

#### Steps to reproduce the issue:
Run script

#### Describe the results you received:
Script run failure due to no pytest topology mark

#### Describe the results you expected:
The script could be run

**Comment by rraghav-cisco** [comment]
@echuawu : the PR 6857 has to do with tests/vxlan/test_vxlan_ecmp.py only. It is marked for t1 already. I have not worked on tests/vxlan/test_vnet_route_leak.py. Pls bring it up with the correct owner. Thanks.

**Comment by echuawu** [comment]
> @echuawu : the PR 6857 has to do with tests/vxlan/test_vxlan_ecmp.py only. It is marked for t1 already. I have not worked on tests/vxlan/test_vnet_route_leak.py. Pls bring it up with the correct owner. Thanks.

Yes, only tests/vxlan/test_vxlan_crm.py is related with this PR.
